packages/cli/src/commands/index.ts

```diff
@@ -382,7 +382,7 @@ export function registerCommands(program: Command) {
     )
     .option(
       '--release <version|next|main>',
-      'Bump to a specific Backstage release line or version',
+      'Bump to a specific Backstage release line or version, or to a specific dist-tag when pattern matches non-@backstage packages',
```

Comment:
```suggestion
      'Bump to a specific Backstage release line or version, or (for third-party packages) to a specific dist-tag',
```

Just workshopping to try to find something a bit clearer and more concise!

Reply:
Perhaps?

> Bump to a specific Backstage release line or version, or to a specific NPM dist-tag

Don't think there's a need to highlight that it's for 3rd party packages right? Should work for `@backstage` packages too right?

Reply:
```suggestion
      'Bump to a specific Backstage release line or version, or to a specific dist-tag',
```

Yeah, I suppose so - just doesn't really make sense for @backstage packages given the existence of release lines. Would be something like this, right?

Reply:
Done - squashed back into existing commit. Took your wording exactly, with the exception of `npm` being [lowercased](https://twitter.com/rockbot/status/896052477841231872).